src/Products/Annotation/Annotator/ArrowAnnotator.cs

```diff
@@ -1,82 +1,93 @@
-﻿using GroupDocs.Annotation.Domain;
-using GroupDocs.Annotation.Domain.Containers;
+﻿using GroupDocs.Annotation.Models;
+using GroupDocs.Annotation.Models.AnnotationModels;
 using GroupDocs.Annotation.MVC.Products.Annotation.Entity.Web;
+using GroupDocs.Annotation.Options;
 using System;
+using System.Globalization;
 
 namespace GroupDocs.Annotation.MVC.Products.Annotation.Annotator
 {
-    public class ArrowAnnotator : AbstractBoxAnnotator
+    public class ArrowAnnotator : BaseAnnotator
     {
-
         private bool withGuid = false;
+        private ArrowAnnotation arrowAnnotation;
 
-        public ArrowAnnotator(AnnotationDataEntity annotationData, PageData pageData)
-            : base(annotationData, pageData)
+        public ArrowAnnotator(AnnotationDataEntity annotationData, PageInfo pageInfo)
+            : base(annotationData, pageInfo)
         {
+            this.arrowAnnotation = new ArrowAnnotation()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove these redundant parentheses.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372470)